config/src/test/java/org/springframework/security/config/doc/XsdDocumentedTests.java

```diff
@@ -65,7 +66,7 @@ public class XsdDocumentedTests {
 
 	String schema31xDocumentLocation = "org/springframework/security/config/spring-security-3.1.xsd";
 
-	String schemaDocumentLocation = "org/springframework/security/config/spring-security-6.4.xsd";
+	String schemaDocumentLocation = "org/springframework/security/config/spring-security-6.5.xsd";
```

Comment:
There are a few more things that need to be done to update the minor version across the project. Will you please leave these changes out and they will be taken care of by a commit that precedes merging this PR.

Reply:
I see. Let's keep it for now so that the build passes. I'll add a note to this ticket to rebase this PR after #16221 is complete.